deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+
+Execute the following command to bring up all services.
+
+```bash
+docker compose up
+docker compose up -d # or run detached
+docker compose logs -t -f # view output from all containers
+```
+
+### MONAI Deploy Services
+- Informatics Gateway - http://localhost:5000 or http://172.29.0.50:5000
+  - SCP Port 104
+- Workflow Manager - http://localhost:5001 or http://172.29.0.60:5001
+- Task Manager - http://localhost:5002 or http://172.29.0.70:5002
+
+### Third-Party Services
+- MinIO (default storage service) - http://localhost:9001 or http://172.29.0.10:9001
+- RabbitMQ (default message broker service) - http://localhost:15672 or http://172.29.0.20:15672
+- MongoDB (default database for Worklfow Manager & Task Manager) - http://localhost:27017 or http://172.29.0.30:27017
+- Orthanc (optional) - http://localhost:8042 or http://172.29.0.100:8042
+  - SCP Port 4242
+
+Note: Orthanc service is included for convenience to demo end-to-end workflow execution. It may be disabled and have MONAI Deploy Lite integrated with external Orthanc, PACS or other DICOM devices,
+
+## Running a MONAI Deploy Workflow
+
+This package includes Orthanc running and connected to the Informatics Gateway, with all required AE Titles pre-configured.
+
+To get started, upload any DICOM dataset to Orthanc. If you don't have any DICOM dataset available, please feel free to download the [Chest CT dataset](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing) or [Abdomen CT dataset](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing). These datasets were converted to DICOM from Medical Decathlon training and validation mages & may be used for the referenced examples later in this document.
+
+### Uploading Data
+
+Navigate to Orthanc via http://localhost:8042 or http://172.29.0.100:8042, and click *Upload* on the top right-hand corner.
+Drag & drop any DICOM files (no folders) to the blue box on the page and then click *Start the upload*.
+
+Or, upload your files to Orthanc using the *storescu* command from [dcmtk](https://dcmtk.org/dcmtk.php.en).
+```
+storescu -v +r +sd -aec ORTHANC localhost 4242 /path/to/my/dicom/*
+```
+
+Navigate to the home page and click *All studies* to confirm data's been uploaded.
+
+### Sample Workflows
+
+Under the **sample-workflows* directory, a couple of sample workflow definitions are provided:
+
+- `hello-world.json `: Hello World!
+- `lung-seg.json`: AI Lung Segmentation MAP
+- `liver-seg.json`: AI Liver Segmentation MAP
+
+
+#### Hello World
+
+##### Description
+
+This example uses the `alpine` image to print all files found in the input directory simply.
+
+1. Deploy the workflow definition to MONAI Deploy Workflow Manager:
+   ```
+   $ curl --request POST --header 'Content-Type: application/json'  --data "@sample-workflows/hello-world.json"  http://localhost:5001/workflows
+
```

Comment:
if we are curling to local host, can we replace all ip address in the env file with localhost?

Reply:
@mocsharp will answer better, but things to understand is that multiple Dockers are run at the same time, communicating on named network, how can one assign local host to all of them?

Reply:
the IPs are for communication between the containers, user shouldn't need to modify them unless they have conflicts.